native/src/components/RemoteContent.tsx

```diff
@@ -37,20 +37,25 @@ type PropType = {
 
 const RemoteContent = (props: PropType): ReactElement => {
   const { onLoad, content, cacheDirectory, theme, resourceCacheUrl, language, onLinkPress } = props
-  const [webViewHeight, setWebViewHeight] = useState(0)
-  const [webViewWidth, setWebViewWidth] = useState(0)
+  // https://github.com/react-native-webview/react-native-webview/issues/1069#issuecomment-651699461
+  const defaultWebviewHeight = 1
+  const [webViewHeight, setWebViewHeight] = useState(defaultWebviewHeight)
+  const [webViewWidth, setWebViewWidth] = useState(defaultWebviewHeight)
+
   useEffect(() => {
-    if (webViewHeight !== 0) {
+    if (webViewHeight !== defaultWebviewHeight) {
       onLoad()
     }
   }, [onLoad, webViewHeight])
```

Comment:
why do we have to watch for changes of the onLoad fkt?

Reply:
If the onLoad func changes for whatever reason, you want to execute it again to apply those changes, In practice, this probably won't happen often though. There is also a recommended eslint rule forcing us to use exhaustive dependencies, i.e. specifying all dependencies of hooks.

Reply:
Its better to add whatever could change, than thinking about it :D